internal/templates/swarm.templ

```diff
@@ -75,20 +72,63 @@ script getChart(chartId string, endpoint string, circleIds []int64) {
 
 
 templ circleInfo(eventInfo data.Event) {
-        <div id="circle-info-container">
-                <pre>{ fmt.Sprintf("%v", eventInfo) }</pre>
-        </div>
+	<div class="relative z-10" aria-labelledby="slide-over-title" role="dialog" aria-modal="true" id="close">
+		<div class="fixed inset-0 bg-gray-500 bg-opacity-75 transition-opacity"></div>
+
+		<div class="fixed inset-0 overflow-hidden">
+			<div class="absolute inset-0 overflow-hidden">
+			<div class="pointer-events-none fixed inset-y-0 right-0 flex max-w-full pl-10">
+
+				<div class="pointer-events-auto relative w-screen max-w-md">
+
+				<div class="absolute left-0 top-0 -ml-8 flex pr-2 pt-4 sm:-ml-10 sm:pr-4">
+					<button type="button" class="relative rounded-md text-gray-300 hover:text-white focus:outline-none focus:ring-2 focus:ring-white" onclick="closePopup('eventInfo.Id')">
+					<span class="absolute -inset-2.5"></span>
+					<span class="sr-only">Close panel</span>
+					<svg class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true">
+						<path stroke-linecap="round" stroke-linejoin="round" d="M6 18L18 6M6 6l12 12" />
+					</svg>
+					</button>
+				</div>
+
+				<div class="flex h-full flex-col overflow-y-scroll bg-white py-6 shadow-xl">
+					<div class="px-4 sm:px-6">
+						<h2 class="text-base font-semibold leading-6 text-gray-900" id="slide-over-title">{ fmt.Sprintf("%v", eventInfo.MergeRequest) }</h2>
+					</div>
+					<div class="relative mt-6 flex-1 px-4 sm:px-6">
+						<pre>Author: { fmt.Sprintf("%v", eventInfo.Actor) }</pre>
+						<a class="text-blue-600" href={ templ.URL(eventInfo.MergeRequestUrl) } target="_blank">{ fmt.Sprintf("%v", eventInfo.MergeRequest) }</a>
+						<pre>Time: { fmt.Sprintf("%v", time.Unix(eventInfo.Timestamp / 1000, 0)) }</pre>
+					</div>
+				</div>
+				</div>
+			</div>
+			</div>
+		</div>
+	</div>
+
+	<script>
+
+		function closePopup(eventId) {
+			document.getElementById('close').style.display = 'none';
+
+			const searchParams = new URLSearchParams(document.location.search);
+			searchParams.delete('event');
+            const newUrl = `${document.location.origin}${document.location.pathname}?${searchParams.toString()}`;
+			history.replaceState(null, '', newUrl);
+		}
```

Comment:
1. What is purpose of eventId ?
2. element id "close" isnt a good name

Reply:
Leftover of the old logic,good you saw that.